IfcReinforcedSoil=>IfcImprovedGround

larswik opened the pull request:
Fixes #384

larswik committed:
IfcReinforcedSoil=>IfcImprovedGround

Fixes #384

SergejMuhic commented on IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Types/IfcImprovedGroundTypeEnum/DocEnumeration.xml:
```diff
@@ -9,6 +9,9 @@
 		<DocConstant xsi:nil="true" href="GROUTED_3h7yaBv192owNn0lteQF65" />
 		<DocConstant xsi:nil="true" href="USERDEFINED_3UZXzKKD12RotG6dmTF8v" />
 		<DocConstant xsi:nil="true" href="NOTDEFINED_0hF15nkf79uy7wMtnpxz" />
+		<DocConstant xsi:nil="true" href="DEEPMIXED_14tVPAsoP2txdXAOYfnl1T" />
```
Move above USERDEFINED.

larswik replied:
Should be fixed now.

SergejMuhic commented on IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcImprovedGround/DocEntity.xml:
```diff
@@ -1,16 +1,16 @@
 	<WhereRules>
 		<DocWhereRule Name="CorrectPredefinedType" UniqueId="e533e402-9d57-4728-be40-35448bd6e3d8">
 			<Documentation>Either the _PredefinedType_ attribute is unset, or the inherited attribute _ObjectType_ shall be provided, if the _PredefinedType_ is set to USERDEFINED.</Documentation>
 			<Expression>NOT(EXISTS(PredefinedType)) OR
```
Looks like changing to HasObjectType again and making PredefinedType mandatory since on IfcObjectType is defined.

larswik replied:
Should be fixed now.

larswik committed:
According to comments

- Moved up new enum constants (before USERDEFINED)
- Made PredefinedType mandatory
- Changed (and renamed) WR

anborr commented on IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Types/IfcImprovedGroundTypeEnum/Documentation.md:
```diff
@@ -0,0 +1 @@
+This container defines the different predefined types (methods) for ground improvement that can specify an _IfcImprovedGround_.
```
```suggestion
This enumeration defines the different predefined types (methods) for ground improvement that can specify an _IfcImprovedGround_.
```

larswik committed:
Update IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Types/IfcImprovedGroundTypeEnum/Documentation.md

Co-authored-by: anborr <73635642+anborr@users.noreply.github.com>

SergejMuhic commented:
Seems @anborr comment was fixed also.
